src/case-swapping.test.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const swap = require('./case-swapping');
+
+test('Should return swapped phrase', () => {
+  expect(swap('HelloWorld'))
+    .toBe('hELLOwORLD');
+});
+
+test('Should return swapped long phrase', () => {
+  expect(swap('lfTm704Unwfis-0acqfnO31 7 _ImaU7-,um.0SIia7.55MJ9'))
```

Comment:
this scenario contains special characters, so it's very similar to the last one. Not sure it brings much value. maybe remove it?